js/ui/windowManager.js

```diff
@@ -1211,6 +1403,22 @@ const WindowManager = new Lang.Class({
         }
     },
 
+    _rotateInCompleted: function(actor) {
+        if (this._removeEffect(this._rotateInActors, actor)) {
+            Tweener.removeTweens(actor);
+            actor.opacity = 255;
+            actor.rotation_angle_x = 0;
+        }
+    },
+
+    _rotateOutCompleted: function(actor) {
+        if (this._removeEffect(this._rotateOutActors, actor)) {
+            Tweener.removeTweens(actor);
+            actor.hide();
```

Comment:
I agree.

At least for now we can probably just expose another d-bus method to pick a PID and "rotate it back" if it was already rotated out. The actual detail of how this is invoked can be determined at some other period. Thoughts?